### PR TITLE
[BUG]: Rename the column RANK since RANK became a reserved keyword for MySQL 8.0.2 and later

### PR DESCRIPTION
**Pull request by AzmiTouil**

### CODE REVIEW

File: services/src/main/resources/conf/portal/configuration.xml [hunk 1]
```diff
@@ -57,6 +57,7 @@
           <value>db/changelog/task.db.changelog-1.2.1.xml</value>
           <value>db/changelog/task.db.changelog-1.3.0.xml</value>
           <value>db/changelog/task.db.changelog-2.1.0.xml</value>
+          <value>db/changelog/task.db.changelog-2.1.1.xml</value>
```
Comment (ahamdi) [inline services/src/main/resources/conf/portal/configuration.xml:60]: This should be 3.0.0 to follow the task application versioning for eXo platform 6.0 IMO
```suggestion
          <value>db/changelog/task.db.changelog-3.0.0.xml</value>
```

Comment (AzmiTouil) [reply services/src/main/resources/conf/portal/configuration.xml]: Done!

File: services/src/main/resources/conf/portal/configuration.xml [hunk 2]
```diff
@@ -57,6 +57,7 @@
           <value>db/changelog/task.db.changelog-1.2.1.xml</value>
           <value>db/changelog/task.db.changelog-1.3.0.xml</value>
           <value>db/changelog/task.db.changelog-2.1.0.xml</value>
+          <value>db/changelog/task.db.changelog-3.0.0.xml</value>
```
Comment (azayati) [inline services/src/main/resources/conf/portal/configuration.xml:60]:
```suggestion
          <value>db/changelog/task.db.changelog-2.1.1.xml</value>
```

Comment (ahamdi) [reply services/src/main/resources/conf/portal/configuration.xml:60]: Well did you see https://github.com/exo-addons/task/pull/237#discussion_r444997356

**Pull request merged**